### PR TITLE
fix: add options to support doing luis and qna cross-train separately

### PR DESCRIPTION
**Pull request by lei9444**
## Description
add crosstrain settings to composer setting:
  inner           Only performs the inner dialog cross train, default is true
  intra            Only performs the intra dialog cross train, default is true

## Task Item
#minor

### CODE REVIEW

**Review by taicchoumsft — APPROVED** [review]

File: Composer/packages/server/src/models/settings/defaultSettingManager.ts [hunk 1]
```diff
@@ -103,6 +104,10 @@ export class DefaultSettingManager extends FileSettingManager {
         isSkill: true,
         allowedCallers: [],
       },
+      crossTrain: {
```
Comment (scheyal) [inline Composer/packages/server/src/models/settings/defaultSettingManager.ts:107]: Terminology: 
Inter --> across
Intra --> Inner/within/internal
So we should have:
Inter / Intra  or 
cross/inner

**Comment by mewa1024** [comment]
(@vivekkshankar fyi)

**Comment by cosmicshuai** [comment]
I tested this PR, the behavior is expected as the result from the bf-cli. Here is a demo
for
```
inner: true
intra: false
```
image
image

for
```
inner: false
intra: true
```
image
image

**Comment by cosmicshuai** [comment]
I tested the latest branch, it works correctly.

**Review by boydc2014 — APPROVED** [review]

**Pull request merged**